observability/opentelemetry-metrics-backend/src/test/scala/sttp/client3/opentelemetry/OpenTelemetryMetricsBackendTest.scala

```diff
@@ -182,6 +182,64 @@ class OpenTelemetryMetricsBackendTest extends AnyFlatSpec with Matchers with Opt
     getHistogramValue(reader, OpenTelemetryMetricsBackend.DefaultLatencyHistogramName).map(_.getSum) should not be empty
   }
 
+  it should "use error counter when http error is wrapped in a try" in {
+    // given
+    val backendStub = SttpBackendStub.synchronous.whenAnyRequest.thenRespondF(_ =>
+      throw HttpError("Not Found", StatusCode.NotFound))
+    val errorBackend = new TryBackend(backendStub)
+    val reader = InMemoryMetricReader.create()
+    val backend = OpenTelemetryMetricsBackend(errorBackend, spawnNewOpenTelemetry(reader))
+
+    // when
+    backend.send(
+      basicRequest
+        .get(uri"http://127.0.0.1/foo")
```

Comment:
to reproduce the issue, shouldn't we have in test a `basicRequest.response(asString.getRight)`, and then the backend stub could simply respond e.g. with a status code 500 (instead of throwing an exception)?

Reply:
OK, I'll change the tests.

Reply:
Should I remove tests with throwing exception?

Reply:
backends don't throw `HttpError`s by themselves as far as I remember, so sucha  test isn't very realistic :)

Reply:
Well then there's no exception that is being thrown, the default response is of type `Either[String, String]` - see https://sttp.softwaremill.com/en/latest/responses/body.html. There's also a description of `.getRight`

Reply:
Done